.github/CODEOWNERS

```diff
@@ -1 +1 @@
-/.github/ @thedevilsvoice
+* @PaloAltoNetworks/gcp-vmseries-modules-maintainers
```

Comment:
Github show error bout it: 
`Unknown owner on line 1: make sure the team @PaloAltoNetworks/gcp-vmseries-modules-maintainers exists and has write access to the repository` it is ok? 
In AzureRM we specify username https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/blob/develop/.github/CODEOWNERS